teuthology/task/install.py

```diff
@@ -927,6 +926,121 @@ def _upgrade_deb_packages(ctx, config, remote, debs):
     )
 
 
+@contextlib.contextmanager
+def rh_install(ctx, config):
+    """
+    Installs rh ceph on all hosts in ctx.
+
+    :param ctx: the argparse.Namespace object
+    :param config: the config dict
+    """
+    version = config['rhbuild']
+    rh_versions = ['1.3.0']
+    if version in rh_versions:
+        log.info("Supported version")
```

Comment:
This isn't a particularly useful message: if you think it's necessary to log this, you might as well include the version in the message